### DISCUSSION TITLE
189

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#162] [General]
# 潮流周刊第189期 - 无印良品

封面图拍摄于家 7 年前买的无印良品的香薰机，中途有几年搬家后被收起来忘记使用，没想到今年一拿出来还可以完美使用，品质比我想的好不少，今年陆续买了不少他家东西均很满意，夏凉被、浴巾都很舒服，不锈钢水杯超级保冷，骨瓷茶壶、茶盘质感很好，甚至把锅铲也换成他家的了，后面计划慢慢生活用品从他家买。

https://weekly.tw93.fun/posts/189-%E6%97%A0%E5%8D%B0%E8%89%AF%E5%93%81

**Comment by Runtu4378** [comment]
无印良品的那个卡片盒，大小跟一个磁吸无线充电宝差不多大。一边可以放4张卡（银行卡、身份证、社保卡），另一边可以放五百块左右的现金零钱。

**Reply by tw93** [reply]
赞 我去看看

**Reply by wuyunyaya** [reply]
Pake  以下操作使用已弃用Node.js版本，并将强制在 node20 上运行：actions/checkout@v3、actions/cache/restore@v3、actions/cache/save@v3、actions/upload-artifact@v3。

**Reply by Runtu4378** [reply]
我用的是这个型号哇 【淘宝】限时满68减3 https://m.tb.cn/h.glSgVwQh2z4xYtR?tk=jqix3e9EU8c HU9196 「新品MUJI无印良品PP卡片盒/双开式双层名片盒聚丙烯名片收纳夹」
点击链接直接打开 或者 淘宝搜索直接打开

**Comment by tonytian2000** [comment]
视频做的真好

**Comment by xxholly32** [comment]
确实，这个视频做的好有创意

**Comment by cMing1997** [comment]
抠图的那个，我还是觉得 remove.bg 比较强，在AI早期发展初期就已经很强了
